release/yaml/ibr_api_v1.0.0.yaml

```diff
@@ -2251,7 +2254,7 @@ components:
         name:
           type: string
           description: |
-            @context: "https://example.org/schema/Attribute" <br>
+            @context: "https://spdci.github.io/schemas//Attribute" <br>
```

Comment:
a minor search + replace problem repeats with the spdci.github.io/schemas// url

Reply:
This will be replaced with domain - https://schema.spdci.org once domain available to me.